packages/ui_v2/src/components/DAOcreatorV2/index.tsx

```diff
@@ -198,6 +209,22 @@ export default function DAOcreator() {
     }
   ];
 
+  const handleMetamask = async () => {
+    try {
+      const address = await getProvider();
+      setDefaultAddress(address);
+      const member = new MemberForm(getDAOTokenSymbol);
+      member.$.address.value = address;
+      member.$.reputation.value = "0";
+      member.$.tokens.value = "0";
+      const firstMember = new MemberForm(member.getDAOTokenSymbol, member);
+      daoForm.$.members.$.push(firstMember);
```

Comment:
Here i'm adding the new member into the `MembersForm` array. But the component `MembersEditor` receives the form as an empty array. I think it should update, since we pass it using props. Am I missing something? @dOrgJelli @mrrobot16